test_programs/cl2/2.lox

```diff
@@ -0,0 +1,15 @@
+---
+expected_error_type: none
+---
+// Multiple instances with a counter
```

Comment:
Let's remove counter, it doesn't test anything specific to the current stage - can be placed in the 4th test if needed, to test unrelated interactions